emulator/blockchain.go

```diff
@@ -1641,3 +1664,35 @@ func (b *Blockchain) GetTransactionResultsByBlockID(blockID flowgo.Identifier) (
 	}
 	return results, nil
 }
+
+func (b *Blockchain) GetSourceFile(location common.Location) string {
+
+	value, exists := b.sourceFileMap[location]
+	if exists {
+		return value
+	}
+
+	addressLocation, isAddressLocation := location.(common.AddressLocation)
+	if isAddressLocation {
```

Comment:
Maybe invert if condition to reduce the amount of nested code.

Reply:
inverted in https://github.com/onflow/flow-emulator/pull/437/commits/5f8fbd3b74599ad16d972bfe25efac4c7854462f